pkg/linter/linter_test.go

```diff
@@ -190,6 +190,96 @@ func Test_optLinter(t *testing.T) {
 	assert.True(t, called)
 }
 
+func Test_pythonMultiplePackagesLinter(t *testing.T) {
+	dir, err := os.MkdirTemp("", "melange.XXXXX")
+	defer os.RemoveAll(dir)
+	assert.NoError(t, err)
+
+	cfg := config.Configuration{
+		Package: config.Package{
+			Name:    "testpythonmultiplepackages",
+			Version: "4.2.0",
+			Epoch:   0,
+			Checks: config.Checks{
```

Comment:
(not blocking) To minimize diffs in future changes, it might be helpful to have a `checkOnly("foo")` helper method in this test package, that returns a `config.Checks` with only one check enabled.

Reply:
yeah. I did this at some point but it's probably gonna need to be a different PR.